docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -5,12 +5,13 @@ box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
 participant ":OneShelfBookParser" as AddressBookParser LOGIC_COLOR
 participant ":ItemDeleteCommandParser" as DeleteCommandParser LOGIC_COLOR
-participant "d:ItemDeleteCommand" as DeleteCommand LOGIC_COLOR
+participant ":ItemDeleteCommand" as DeleteCommand LOGIC_COLOR
```

Comment:
Having the d, or some label there, might be usefeul, to show the command being passed back to the front. For instance, right now you do use the "command" label on one of the return arrows, which essentially represents the "d" here

Reply:
fixed in latest commit, used 'command' instead of 'd'
https://github.com/AY2021S1-CS2103T-T12-1/tp/pull/169/commits/c276c055ccc67713af31fe5506ca399aa0cee012